src/qmapshack/setup/CAppSetupWin.cpp

```diff
@@ -28,7 +28,7 @@ void CAppSetupWin::initQMapShack() {
   QString gdalDir = QString("%1\\data").arg(apppath);
   QString projDir = QString("%1\\share\\proj").arg(apppath);
 
```

Comment:
Is it possible to set `GDAL_DRIVER_PATH` in this way? 
```suggestion
 
 QString gdalpluginsDir = QString("%1\\gdalplugins").arg(apppath);
  qputenv("GDAL_DRIVER_PATH", gdalpluginsDir);

Reply:
To give some context, the proposal came up when we were testing the new installer.
In theory and by default gdal should load all the drivers included in the gdalplugins folder, but for a reason we didn't catch it was ignoring it. 
So it was necessary to set the GDAL_DRIVER_PATH variable before starting QMS,
 One way would be this proposal (but I'm not sure if the code is well written so I asked, and it also should be tested ).
 The other is with a bat file that sets the variable and then starts QMS, which is the one chosen now by @wthaem  (and I like it too).

I consider both options to be valid, and in principle they are not mutually exclusive. The advantage of the first one is that the .bat is not essential.
In the case of the second one, @wthaem has already changed the shortcuts in the win menu pointing to the .bat instead of the exe. The only doubt is that if a user already has a  previous shortcut on the desktop pointing to the exe, he would have to overwrite it, or may be an option in the installer to do it could be included.

Reply:
Imho doing it in the batch file is the correct way to do it. Ideally we wouldn't need that stuff because everything has it's well defined place to go, e.g. in Linux everything has to install it's stuff into well defined global paths. But this is Windows. Everything is hidden in it's own proprietary path and the environment variables are pointing to the right location. As the GDAL stuff is private to QMS we can't set the variables globally. We have to set them locally in the batch. By that we can change it if necessary without changing the code.